packages/components/menu/src/base/menu-item-base.tsx

```
@@ -88,11 +88,18 @@ interface Props<T extends object = {}> extends Omit<ItemProps<"li", T>, "childre
   classNames?: SlotsToClasses<MenuItemSlots>;
 }
 
-export type MenuItemBaseProps<T extends object = {}> = Props<T> &
+export type MenuItemBaseProps<T extends object = {}> = Omit<Props<T>, "onClick"> &
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)